docs/UserGuide.md

```diff
@@ -221,15 +250,16 @@ Use the `list /e` command to view the indices of your expenditures.
     * Enter the amount in dollars, rounded off to the nearest cent.
     * For example: an item that cost 420 dollars and 69 cents will be entered as `420.69`.
 * `[NEW_DATE]` refers to the new date of the purchase of the expenditure.
-    * Format of the date is DD/MM/YYYY.
+    * Format of the date is `dd/mm/yyyy`.
     * For example: `12 March 2022` will be entered as `12/03/2022`.
 
 For example: `update /e 1 /pm cash /c Food /d chicken rice /a 5 /t 12/03/2022`.  
 Updates the first expenditure in your list to a $5.0 expenditure on food item 'chicken rice' that was paid in cash on 12
 March 2022.
 
 #### Expected Outcome:
-
+Update the first expenditure in your list to a $5.0 expenditure on food item 'chicken rice' that was paid in cash on 12
```

Comment:
why include this here when its mentioned above in line 257?

Reply:
resolved.